Unix/tests/cli/test_cli.cpp

```diff
@@ -74,6 +75,100 @@ namespace
     bool runKrbTests       = false;
 }
 
+
+
+// Post Mortem log handling. Maybe should be in nits base. 
```

Comment:
I am confused.  I thought we'd dump the log for the whole travis run, not just for cli tests.  This would help for preexec, but not if failure occurs in wsman, protocol, etc.?

Reply:
When the code runs correctly we can reshuffle a bit. (for the next release)